src/legacy/core_plugins/vis_type_metric/public/__tests__/metric_vis_controller.js

```diff
@@ -51,7 +51,7 @@ describe('metric_vis - controller', function () {
 
     expect(metrics.length).to.be(1);
     expect(metrics[0].label).to.be('Count');
-    expect(metrics[0].value).to.be(4301021);
+    expect(metrics[0].value).to.be('<span ng-non-bindable>4301021</span>');
```

Comment:
Why did this change occur?

Reply:
Looking at a line 124 at a `metric_vis_controller.js` component it seems to be fine.